third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -545,11 +563,16 @@ class BlockedToMFMA : public OpRewritePattern<tt::DotOp> {
         chooseMfmaInstruction(dotOp, mfmaVersion, nonKDim, withScale);
     if (failed(mfmaInstr)) {
       if (!withScale) {
+        dotOp.emitRemark()
+          << "Unable to choose double-rated MFMA intrinsic for dot operation.";
```

Comment:
It's not specifically to "double-rated" actually? We can drop that.

Reply:
Yes it's trying to choose a preferable variant. Though "double-rate" can be confusing here as part of the error message given we don't have any such formal definition. So we should drop it, or maybe change to say "preferable" or something.